### ISSUE TITLE
[asyncio] Typo in variable name

### ISSUE THREAD

**Issue by Solomon1732** [Tribler/py-ipv8#638]
In [this](https://github.com/Tribler/py-ipv8/blob/babcdae067de773526ef25eb82eef76b35268f37/scripts/trustchain_crawler_plugin.py#L157) the lambda holds a parameter named `sign`, yet it uses `sig` inside the lambda.
```python
signal.signal(signal.SIGTERM, lambda sign, _: ensure_future(signal_handler(sig)))
```
Also the line right above it has `sig` as its parameter name instead of `sign`. In addition `sig` is used in the rest of the function. Is it a typo?

Running flake8 detects `sig` in the line as undefined.

**Comment by qstokkink** [comment]
Not a typo, confirmed bug. Thanks for reporting.

Either the `lambda` should define `sig` or the `signal_handler` should take `sign`, probably the former.

**Comment by Solomon1732** [comment]
Looks like a quick fix. I can PR it if you want. Not sure to what branch, though.

**Comment by qstokkink** [comment]
That would be very much appreciated, this is on the `asyncio` base branch.

**Comment by Solomon1732** [comment]
Fixed in #639 . Closing the issue
